app/daemon.go

```diff
@@ -131,14 +131,16 @@ func startManager(c *cli.Context) error {
 		return err
 	}
 
-	ds, wsc, err := controller.StartControllers(logger, done, currentNodeID, serviceAccount, managerImage, kubeconfigPath, meta.Version)
+	proxyStore := util.NewProxyStore()
```

Comment:
The naming probably can be improved to understand. How about `proxyConnCounter` ?

Reply:
Updated.